src/Model/SeasonManager.php

```diff
@@ -21,19 +21,33 @@ public function __construct ()
         parent::__construct(self::TABLE);
     }
 
+    /**
+     * @param array $data
+     * @return string|void
+     * @throws \Exception
+     */
     public function insert (array $data)
     {
         $idSerie = $data[ 'idSerie' ];
-        unset($data[ 'idSerie' ]);
         $nb = $data[ 'nbSeasons' ];
-        $query = "SELECT max(numero_season) FROM $this->table WHERE idserie = $idSerie";
-        $result = $this->pdoConnection->query($query, \PDO::FETCH_ASSOC)->fetch();
-        $lastSeason = $result[ 'max(numero_season)' ];
-        $query = "INSERT INTO $this->table (numero_season, idserie) VALUES (:numb, :idSerie)";
-        $statement = $this->pdoConnection->prepare($query);
-        $statement->bindValue('numb', $lastSeason + $nb);
-        $statement->bindValue('idSerie', $idSerie);
-        $statement->execute();
+        $verif = "SELECT numero_season FROM $this->table WHERE idserie = $idSerie";
```

Comment:
SELECT numero_season FROM $this->table WHERE numero_season=:num_season AND idSerie=:idSerie